Source/SwiftLintFramework/Rules/UnusedClosureParameterRule.swift

```diff
@@ -93,6 +106,8 @@ public struct UnusedClosureParameterRule: ASTRule, ConfigurationProviderRule, Co
                 return nil
             }
 
+            let paramLength = name.characters.count
```

Comment:
Either approach causes SourceKitten to crash [here](https://github.com/jpsim/SourceKitten/blob/b12e1442854eb6d283a4f7e7c09f2207d1652906/Source/SourceKittenFramework/String+SourceKitten.swift#L131). I'll look into why tomorrow.